portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase

```diff
@@ -8,78 +8,86 @@ definition {
 	setUp {
 		TestCase.setUpPortalInstance();
 
-		User.firstLoginPG();
+		JSONGroup.addGroup(groupName = "Test Site Name");
 
-		Navigator.openURL();
-
-		ApplicationsMenu.gotoPortlet(
-			category = "Configuration",
-			panel = "Control Panel",
-			portlet = "Instance Settings");
+		task ("Disable email verification for strangers") {
```

Comment:
can you do it by json call?

Reply:
Probably but for this PR I will leave it like this. I'll try to refactor this again in the future.

Reply:
ok